zulipterminal/ui_tools/views.py

```diff
@@ -839,6 +839,19 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
             self.controller.exit_popup()
         return super(PopUpConfirmationView, self).keypress(size, key)
 
+class StreamInfoView(urwid.ListBox):
+    def __init__(self, controller: Any, color: Any, name: Any, desc: Any) -> None:
+        self.controller = controller
+        self.width = max(len(desc), len(name))
+        self.height = 3
+        self.name = urwid.Text((color, name), align='center')
+        self.desc = urwid.Text(desc, align='center')
```

Comment:
For these objects, do we need all of these as attributes, or are they just names manipulated and passed elsewhere?